examples/grpc/grpc-streaming-client/main.go

```diff
@@ -0,0 +1,241 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"gofr.dev/examples/grpc/grpc-streaming-client/client"
+	"gofr.dev/pkg/gofr"
+	"io"
+	"time"
```

Comment:
Fix imports order please

Reply:
I'm surprised golangci-lint didn't catch this.

Reply:
It comes from the setting `issues.exclude-dirs-use-default` that is set to true by default, and that excludes `examples` folder to be linted.